internal/packages/internal/packagerender/objects_test.go

```diff
@@ -106,6 +106,7 @@ func TestFilterWithCELAnnotation(t *testing.T) {
 			if tc.err == "" {
 				require.NoError(t, err)
 				require.Equal(t, len(tc.filtered), len(filtered))
+				require.Equal(t, tc.filteredIndexes, filteredIndexes)
```

Comment:
I'm curious: Why is this line added in this PR?

Reply:
Oh nevermind! GH showed it as changed, but this was "the change between my old review and this review"!